.clang-format

```diff
@@ -121,5 +98,5 @@ SpacesInParentheses: false
 SpacesInSquareBrackets: false
 Standard:        Cpp11
 TabWidth:        8
```

Comment:
Should TabWidth be set to 4 just for consistency? Or should we remove this explicit setting if we aren't using it?

Reply:
Sure. I'll most likely set it to 4 since I'd rather have the full list of settings so it's easier to edit/compare.